backend/elasticsearch/src/main/java/com/rbmhtechnology/vind/elasticsearch/backend/util/ResultUtils.java

```diff
@@ -103,13 +112,25 @@ public static FacetResults buildFacetResults(Aggregations aggregations, Document
             .map(intervalFacet -> getIntervalFacetResults(aggregationsMap.get(Stream.of(searchContext, intervalFacet.getFacetName()).filter(Objects::nonNull).collect(Collectors.joining("_"))), (Facet.IntervalFacet)intervalFacet))
             .forEach(pair -> intervalFacetResults.put(pair.getKey(), pair.getValue()));
 
-//TODO:
         HashMap<String, StatsFacetResult<?>> statsFacetResults = new HashMap<>();
```

Comment:
for the matter of readability it would be great to extract the method and add a short comment, what actually happens. Streams are hard to review...

Reply:
I was think about a refactoring, splitting files based on the Vind concepts: Facet, filters suggestions,... instead of all mixed together in a build query  file and build result file as we have for solr backend. And having also this splitting of the streams in mind.